uberfire-extensions/uberfire-wires/uberfire-wires-core/uberfire-wires-core-grids/src/main/java/org/uberfire/ext/wires/core/grids/client/widget/layer/impl/GridLienzoPanel.java

```diff
@@ -83,6 +83,38 @@ public void onResize() {
                         height);
     }
 
+    public GridLienzoPanel(final DefaultGridLayer defaultGridLayer) {
+        this(new LienzoPanel() {
+                 @Override
+                 public void onResize() {
```

Comment:
If I remember correctly in Test Scenario we use a `onResize()` to force rendering so I don't know if this could break that (not sure it is the same `onResize()` method).
@gitgabrio ^^

Reply:
@manstis @danielezonca 
Yes, we call `onResize` to force redrwan of the grid, IIRC

Reply:
The 2 new constructors are no different to the 2 existing constructors (one of which _scesim_ would be using). Well, when I say no different, I mean their use of `LienzoPanel` is no different and they simply allow me to pass a `DefaultGridLayer`. I see this to have no affect on _scesim_.